lib/tpm2_session.h

```diff
@@ -32,6 +32,9 @@
 
 #include <tss2/tss2_sys.h>
 
+#define MAX_AUTH_SESSIONS 3
```

Comment:
this is the ARRAY_LEN() of type TSS2L_SYS_AUTH_COMMAND AUTHS field... so one could use that.